.github/workflows/build.yml

```diff
@@ -0,0 +1,54 @@
+name: Build
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+env:
+  DOTNET_NOLOGO: true
```

Comment:
You might also want to add `DOTNET_CLI_TELEMETRY_OPTOUT: 1` to prevent the dotnet cli from phoning home.

Reply:
done